kubernetes/README.md

```diff
@@ -15,17 +15,21 @@ it creates a ConfigMap and a CoreDNS deployment, then updates the Kube-DNS servi
 to use the CoreDNS deployment. By re-using the existing service, there is no disruption in
 servicing requests.
 
+By providing the `-u` option, the deployment script will translate the existing kube-dns configuration
+into the equivalent CoreDNS Corefile.
```

Comment:
I thought we said that the default behavior was to automatically upgrade from kube-dns
and the option to avoid that upgrade and just provide the standard ... 

Although, doing with -u is compatible with existing behavior.